pyrolite/util/classification.py

```diff
@@ -289,6 +289,110 @@ def add_to_axes(self, ax=None, fill=False, axes_scale=100.0, labels=None, **kwar
         ax.set_xlabel("$SiO_2$")
         return ax
 
+class AIOCG(PolygonClassifier):
+    """
+    AIOCG Diagram classifier from Montreuil et al. (2013) [#ref_1]_.
+
+    Parameters
+    -----------
+    name : :class:`str`
+        A name for the classifier model.
+    axes : :class:`list` | :class:`tuple`
+        Names of the axes corresponding to the polygon coordinates.
+    fields : :class:`dict`
+        Dictionary describing indiviudal polygons, with identifiers as keys and
+        dictionaries containing 'name' and 'fields' items.
+    scale : :class:`float`
+        Default maximum scale for the axes. Typically 100 (wt%) or 1 (fractional).
+    xlim : :class:`tuple`
+        Default x-limits for this classifier for plotting.
+    ylim : :class:`tuple`
+        Default y-limits for this classifier for plotting.
+
+    References
+    -----------
+    .. [#ref_1] Montreuil J F, Corriveau L, and Potter E G (2015). Formation of 
+            albitite-hosted uranium within IOCG systems: the Southern Breccia, 
+            Great Bear magmatic zone, Northwest Territories, Canada. 
+            Mineralium Deposita, 50:293-325.
+            doi:`<https://doi.org/10.1007/s00126-014-0530-7>`__
+    """
+
+    @update_docstring_references
+    def __init__(self, **kwargs):
+        src = pyrolite_datafolder(subfolder="models") / "AIOCG" / "config.json"
+
+        with open(src, "r") as f:
+            config = json.load(f)
+        kw = dict(scale=100.0, xlim=[0, 607.4], ylim=[0, 405.2])
+        kw.update(kwargs)
+        poly_config = {**config, **kw}
+        super().__init__(**poly_config)
+
+    def add_to_axes(self, ax=None, fill=False, axes_scale=100.0, labels=None, **kwargs):
+        """
+        Add the AIOCG fields from the classifier to an axis.
+
+        Parameters
+        ----------
+        ax : :class:`matplotlib.axes.Axes`
+            Axis to add the polygons to.
+        fill : :class:`bool`
+            Whether to fill the polygons.
+        axes_scale : :class:`float`
```

Comment:
I think the default scale for this is `1.0`; and then you'll want to set the aspect of the `Axes` to 6:4?